### PR TITLE
Clippy warnings

### PR DESCRIPTION
**Pull request by EPashkin**
cc @GuillaumeGomez, @sdroege

### CODE REVIEW

**Comment by EPashkin** [comment]
MacOs nightly error first appeared in https://travis-ci.org/gtk-rs/cairo/jobs/390374585 for https://github.com/gtk-rs/cairo/pull/191
and we think that it futures-related, but seems - not.

Previous PR's job is https://travis-ci.org/gtk-rs/cairo/jobs/380392079 passed.

**Comment by EPashkin** [comment]
I don't see any between this jobs, maybe just try restart failed?

**Comment by EPashkin** [comment]
It passed now, strange

File: cairo-sys-rs/src/enums.rs [hunk 1]
```diff
@@ -70,9 +70,9 @@ impl Debug for Status {
 }
 
 impl Status {
-    pub fn ensure_valid(&self) {
-        if *self != Status::Success {
-            panic!("Cairo error {:?}", *self)
+    pub fn ensure_valid(self) {
```
[the remark this answers is not in the document]
Comment (EPashkin) [reply cairo-sys-rs/src/enums.rs]: Enums is Copy

[the remark this answers is not in the document]
Comment (GuillaumeGomez) [reply cairo-sys-rs/src/enums.rs]: Solves issue. :D

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**